### PR TITLE
Fetching data

### PR DESCRIPTION
**Pull request by saladq**
completed week 5 tasks.

### CODE REVIEW

**Comment by phuongnguyen00** [comment]
The code looks good to me! Very clear and concise.

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -15,9 +15,88 @@
...
+
+      // checking for open times to schedule meeting
+      for(int i=1; i<attendeeTimes.size(); i++){
+        TimeRange time = attendeeTimes.get(i);
+        if (!time.overlaps(latestTime)){
+            if(time.start()-latestTime.end() >= requestDuration){
+                TimeRange timeOpen = TimeRange.fromStartDuration(latestTime.end(), time.start()-latestTime.end());
+                res.add(timeOpen);
+            }
+        }
```
Comment (phuongnguyen00) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:88]: Is it assumed that the times being iterated over are disjoint?

Comment (saladq) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:88]: not exactly but I always check to make sure they don't overlap.

**Review by mariamaynard — APPROVED** [review]
I think it looks good to me.

**Pull request merged**